src/fparser/one/tests/test_parsefortran.py

```diff
@@ -98,6 +98,7 @@ def next(self):
                             'warning':  [],
                             'error':    [],
                             'critical': []}
+    _.cache.clear()
```

Comment:
We only use `_` if we don't care about whatever is being stored. As we do now reuse it in order to clear the cache, please could you give it a name, e.g. `parser = fparser.one.parsefortran.....` and then this line will be `parser.cache.clear()`.